### PR TITLE
168167685-corrected-highsore

### PR DESCRIPTION
**Pull request by elvisiraguha**
- set to check if highscore is NaN instead of checking if its null

[finishes #168167685]

### CODE REVIEW

File: src/score.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ export default class Score {
   }
 
   saveHighestScore(score: number) {
-    if (score > this.highestScore || this.highestScore === null) {
+    if (score > this.highestScore || isNaN(this.highestScore)) {
       localStorage.setItem('highestScore', score);
-      this.displayHighestScore = score;
+      this.displayHighestScore(score);
```
Comment (Nziranziza) [inline src/score.js]: with setters, we don't call a function as other function. we assign it a value like a variable

Comment (elvisiraguha) [reply src/score.js]: okay let me change it

**Comment by Nziranziza** [comment]
Add a description to your PR
Always add commit message

**Comment by elvisiraguha** [comment]
> Add a description to your PR
> Always add commit message

done

**Comment by Nziranziza** [comment]
Great job @elvisiraguha !

**Pull request merged**